Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM docker.stackable.tech/stackable/hadoop:3.3.6-stackable0.0.0-dev
 
 COPY --chown=stackable:stackable ./hdfs-utils-*.jar /stackable/hadoop/share/hadoop/tools/lib/
+COPY --chown=stackable:stackable ./bom.json /stackable/hadoop/share/hadoop/tools/lib/hdfs-utils.cdx.json
```

Comment:
Is this really the best location to place the bom? Just curious, no opinion from me

Reply:
I can think of two options:
a) Place the BOM "near" the component it's for, that's how it's done right now
b) Place all BOMs in the same directory, like /stackable/sboms/

I slightly prefer option a) because I feel it clearer states "this SBOM is for this component right here". If, for some reason, the directory is copied to somewhere else (or packed into an archive) then the BOM will always be included. Also, that's how we do it for our operators and config-utils at the moment. No strong opinion though, we can easily change this if it creates any problems at some point in the future.

Reply:
Makes sense, thanks!